core/src/syn/parser/stmt/parts.rs

```diff
@@ -107,8 +107,11 @@ impl Parser<'_> {
 		if !self.eat(t!("FETCH")) {
 			return Ok(None);
 		}
-		let v = self.parse_idiom_list(ctx).await?.into_iter().map(Fetch).collect();
-		Ok(Some(Fetchs(v)))
+		let mut fetchs = vec![Fetch(self.parse_value_field(ctx).await?)];
+		while self.eat(t!(",")) {
+			fetchs.push(Fetch(self.parse_value_field(ctx).await?));
+		}
+		Ok(Some(Fetchs(fetchs)))
```

Comment:
This will parse all possible values after fetch, including subqueries, blocks statements etc. This should be more restricted, testing for a parameter only, otherwise parsing an idiom.